app/main/plugins/hain-plugin-math/index.js

```diff
@@ -29,22 +32,50 @@ module.exports = (context) => {
 
   function makeResultItem(titleKey, query, answer) {
     const result = {};
-    result[titleKey] = `${query.trim()} = ${answer}`;
+    result[titleKey] = `${query.trim()} = <b>${answer}</b>`;
     result.group = 'Math';
     result.payload = answer;
     return result;
   }
 
-  function calculate(query, showRedundantResult) {
+  function calculate(rawQuery, showRedundantResult) {
     try {
+      let query = rawQuery;
+
+      // determine input mode by the presence of European-style decimal separators
```

Comment:
Why not use Intl.NumberFormat?

`console.log((new Intl.NumberFormat()).resolvedOptions());`
Shows this on my system (I believe it gets it from the system)
> { locale: 'und',
>   numberingSystem: 'latn',
>   style: 'decimal',
>   useGrouping: true,
>   minimumIntegerDigits: 1,
>   minimumFractionDigits: 0,
>   maximumFractionDigits: 3 }

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

Reply:
@cpriest it's an interesting suggestion, but I prefer my solution as:
* it is simpler, and more likely to work across platforms
* it allows users to utilise either decimal point or comma independently of whatever the system locale might be set to

Reply:
@cpriest are you happy for this to be merged as-is?